OpenOversight/app/models/database.py

```diff
@@ -341,6 +338,17 @@ def remove_database_cache_entries(self, update_types: List[str]) -> None:
         """Remove the Department model key from the cache if it exists."""
         remove_database_cache_entries(self, update_types)
 
+    @staticmethod
+    def by_state() -> dict[str, list["Department"]]:
```

Comment:
`ist["Department"]`: Perhaps this is how we can update the typing on the other static methods.

Reply:
Updated typing